src/ConcreteOperations/minkowski_sum.jl

```diff
@@ -389,9 +395,17 @@ function minkowski_sum(P1::VPolytope, P2::VPolytope;
 
     vlist1 = _vertices_list(P1, backend)
     vlist2 = _vertices_list(P2, backend)
+    Vout = _minkowski_sum_vrep_nd(vlist1, vlist2, apply_convex_hull=apply_convex_hull, backend=backend, solver=solver)
+    return VPolytope(Vout)
+end
+
+function _minkowski_sum_vrep_nd(vlist1::Vector{VT}, vlist2::Vector{VT};
```

Comment:
```suggestion
function _minkowski_sum_vrep_nd(vlist1::AbstractVector{VT}, vlist2::AbstractVector{VT};
```

Reply:
i think the `Vector` of vectors covers the common cases.. ?

Reply:
Does the implementation rely on anything that is specific to `Vector`? If not, I think it should not have this restriction.

Reply:
ok. i think the implementation doesn't rely specifically an outer `Vector`; i just assumed that it is the case because it is the usual case. but i can change it